packages/core/components/Checkbox/index.tsx

```diff
@@ -25,6 +26,8 @@ export default function BaseCheckbox(props: Props) {
     }
     return (
         <Checkbox
+            // defaults to undefined (handles its own state)
```

Comment:
What here defaults to undefined? Isn't `isChecked` a boolean?

Reply:
This was referring to the internal FluentUI component (not ours), which doesn't require the `checked` property so defaults to undefined when that isn't provided. I think my comment is more confusing than clarifying though so I'll take it out!